src/inc/MSIXResource.hpp.cmakein

```diff
@@ -0,0 +1,56 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+//
+//  This file is generated by CMake. Do not edit.
+//
+#include "AppxPackaging.hpp"
+#include "ComHelper.hpp"
+#include "AppxFactory.hpp"
+#include <map>
+
+namespace MSIX {
+
+    ${NAMESPACE_MANAGER}

```

Comment:
It feels like this should get dumped into its own cpp with an extern reference here.